dipper/curie_map.yaml

```diff
@@ -147,7 +147,7 @@
 
 # variants / traits
 
-# 'AQTL-BT': https://www.animalgenome.org/cgi-bin/QTLdb/BT/search       # maybe?
+# 'AnimalQTLdb': 'previously'
```

Comment:
Is this intentional?

Reply:
Yes, 'AnimalQTLdb' existed as a curie prefix in the past.
it  was broken and did not go anywhere useful but   
I wanted to leave a breadcrumb in case anyone ever came looking for it.